config/initializers/bypass_ssl_verification.rb

```diff
@@ -0,0 +1,2 @@
+require 'openssl'
+OpenSSL::SSL::VERIFY_PEER = OpenSSL::SSL::VERIFY_NONE
```

Comment:
pourkeuwa ça ?

Reply:
j'ai un bug lié a mac : l'api mango pay ne fonctionne pas si je ne desactive pas le ssl
https://www.redmineup.com/pages/help/helpdesk/how-to-fix-issues-with-ssl-certificate